fast_barcode_scanner/lib/src/camera_controller.dart

```diff
@@ -154,27 +154,33 @@ class _CameraController implements CameraController {
   /// User-defined handler, called when a barcode is detected
   OnDetectionHandler? _onScan;
 
+  OnDetectionHandler _buildScanHandler(OnDetectionHandler? onScan) {
```

Comment:
Small doc mentioning that this is a curried function for onScan.

Reply:
will fix.